CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 # Changelog
+## v1.1.0 (2025-03-03)
```

Comment:
@ivana-truong Just a reminder to update this date when things are all ready to go